DataManagementSystem/Client/DataManager.py

```diff
@@ -662,6 +662,8 @@ def __replicate( self, lfn, destSE, sourceSE = '', destPath = '', localCache = '
         self.getFile( lfn, destinationDir = localDir )
         localFile = os.path.join( localDir, os.path.basename( lfn ) )
         fileDict = {destPfn:localFile}
+        #FIXME: This is to avoid third party transfer attempt by the SE, will be better treated in the future
```

Comment:
okay, I just hope it doesn't break any other SE plugin. It should be okay for srm and DIP from what I can see